client/cmd/root.go

```diff
@@ -52,6 +54,8 @@ var (
 	natExternalIPs          []string
 	customDNSAddress        string
 	rosenpassEnabled        bool
+	interfaceName           string
+	wireguardPort           int
```

Comment:
Can we make this `uint16` everywhere to limit the possible values naturally?

Reply:
good catch. I've pushed the check